sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClientConstants.cs

```diff
@@ -16,6 +16,7 @@ internal class ManagementClientConstants
         public const string SbNs = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect";
         public const string XmlSchemaInstanceNs = "http://www.w3.org/2001/XMLSchema-instance";
         public const string XmlSchemaNs = "http://www.w3.org/2001/XMLSchema";
+        public const string SerializationNamespace = "http://schemas.microsoft.com/2003/10/Serialization/";
```

Comment:
nit:  The other members of this file use `Ns` where this uses `Namespace`; maybe we should consider unifying on one representation?   Personally, my preference goes to `Namespace`

Reply:
I'm good with either @jsquire. Fully spelled out version is easier to read though. So if @nemakam agrees, I'll update the other two constant names despite being out of the context.

Reply:
I am good with either.
@SeanFeldman, you can take the call to go either way :)

Reply:
Will be addressed in https://github.com/Azure/azure-sdk-for-net/pull/7399